src/main/java/duke/tasks/ToDo.java

```diff
@@ -0,0 +1,39 @@
+package duke.tasks;
+
+/**
+ * The ToDo class represents a to-do task in the Duke chatbot application.
+ * It extends the Task class.
+ */
+public class ToDo extends Task {
+
+    private static final String TODO_MESSAGE = "[T]%s";
+    private static final String TODO_FILE_TEMPLATE = "T | %s";
+
+    /**
+     * Constructs a ToDo object with the specified description.
+     *
+     * @param description The description of the to-do task.
+     */
+    public ToDo(String description) {
+        super(description);
+    }
+
+    /**
+     * Returns the string representation of the to-do task for saving to a file.
+     *
+     * @return The string representation of the to-do task for saving to a file.
+     */
```

Comment:
Should `@return` be omitted, since the return value is obvious from the rest of the comment?